sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -148,14 +148,41 @@ def configure_span_processors
       end
 
       def default_span_processor
-        Trace::Export::SimpleSpanProcessor.new(
+        Trace::Export::SimpleSpanProcessor.new(configured_exporter)
```

Comment:
>The default span processor is not specified and there is no specified environment variable to select it. We defaulted to the SimpleSpanProcessor previously, and I've kept that default here, but that is rarely useful with any of the defined exporters.

Does it make sense to just give users a better default then?  I recall at least one user reporting that they went with the simple span processor in production and had issues as a result.

Reply:
> Does it make sense to just give users a better default then? I recall at least one user reporting that they went with the simple span processor in production and had issues as a result.

Yes, I've chosen to wrap the fallback exporter (the console span exporter) in a SimpleSpanProcessor and wrap all others in a BatchSpanProcessor.

Reply:
^ i think this is the appropriate approach